examples/cloud-integrations-gcp.tf

```diff
@@ -0,0 +1,69 @@
+/*
+
+    Complete example to enable AWS integration with New Relic
+
+*/
+
+
+variable "NEW_RELIC_SERVICE_ACCOUNT_ID" {
+  type = string
+}
+
+variable "GCP_PROJECT_ID" {
+  type = string
+}
+
+variable "NR_ACCOUNT_ID" {
+  type = string
+}
+
+resource "google_project_iam_member" "project" {
+  project = var.GCP_PROJECT_ID
+  role    = "roles/editor"
+  member  = "serviceAccount:${var.NEW_RELIC_SERVICE_ACCOUNT_ID}"
+}
+
+resource "google_project_iam_binding" "project" {
+  project = var.GCP_PROJECT_ID
+  role    = "roles/editor"
+
+  members = [
+    "serviceAccount:${var.NEW_RELIC_SERVICE_ACCOUNT_ID}",
+  ]
+}
+
+resource "newrelic_cloud_gcp_link_account" "gcp_account" {
+  account_id=var.NR_ACCOUNT_ID
```

Comment:
Formatting looks a little off here. Besides that it looks good to me.

Reply:
code formatted